### PR TITLE
AGDIGGER-206 docs for digger node client

### PR DESCRIPTION
**Pull request by aliok**
Copy the generated JSDocs generated in Digger node client project and add references.

### CODE REVIEW

**Comment by aliok** [comment]
cc @laurafitzgerald @StephenCoady @PhilipGough

**Comment by laurafitzgerald** [comment]
@aliok :eyes:

File: docs/specs/aerogear-digger-node-client/api_jenkinsInfo.js.html [hunk 1]
```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
```
Comment (laurafitzgerald) [inline docs/specs/aerogear-digger-node-client/api_jenkinsInfo.js.html]: @aliok I don't see that this file is linked anywhere?

File: docs/specs/aerogear-digger-node-client/index.html [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+    </section>
+
+
+
+
+
+
+</div>
+
+<nav>
```
Comment (laurafitzgerald) [inline docs/specs/aerogear-digger-node-client/index.html]: I think the links are a bit funny here: Could we change `Global` to a heading like `Modules` and have a link underneath it?

File: docs/specs/aerogear-digger-node-client/scripts/prettify/Apache-License-2.0.txt [hunk 3]
```diff
@@ -0,0 +1,202 @@
+
```
Comment (laurafitzgerald) [inline docs/specs/aerogear-digger-node-client/scripts/prettify/Apache-License-2.0.txt]: Why are we ading the Licence here?

File: docs/specs/aerogear-digger-node-client/global.html [hunk 4]
```diff
@@ -0,0 +1,285 @@
...
+    <div class="container-overview">
+    
+        
+
+        
+
+
+<dl class="details">
+
+
```
Comment (laurafitzgerald) [inline docs/specs/aerogear-digger-node-client/global.html]: lots of space here?

File: docs/specs/aerogear-digger-node-client/util_sslSetup.js.html [hunk 5]
```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
```
Comment (laurafitzgerald) [inline docs/specs/aerogear-digger-node-client/util_sslSetup.js.html]: also not a link to this file?

**Comment by laurafitzgerald** [comment]
@aliok left some comments/questions in-line.

File: docs/specs/aerogear-digger-node-client/api_jenkinsInfo.js.html (hunk 1 above)
Comment (aliok) [reply docs/specs/aerogear-digger-node-client/api_jenkinsInfo.js.html]: @laurafitzgerald that is linked in the JSDocs generated. I advise this:

* do the steps to run the AG.org website locally
* go to localhost:4000
* go to Digger page
* click on Digger Node Client specs button
* you will see the JSDoc HTML generated

File: docs/specs/aerogear-digger-node-client/index.html (hunk 2 above)
Comment (aliok) [reply docs/specs/aerogear-digger-node-client/index.html]: this is generated

File: docs/specs/aerogear-digger-node-client/scripts/prettify/Apache-License-2.0.txt (hunk 3 above)
Comment (aliok) [reply docs/specs/aerogear-digger-node-client/scripts/prettify/Apache-License-2.0.txt]: this is generated

File: docs/specs/aerogear-digger-node-client/global.html (hunk 4 above)
Comment (aliok) [reply docs/specs/aerogear-digger-node-client/global.html]: this is generated

File: docs/specs/aerogear-digger-node-client/util_sslSetup.js.html (hunk 5 above)
Comment (aliok) [reply docs/specs/aerogear-digger-node-client/util_sslSetup.js.html]: similar to above.

File: docs/specs/aerogear-digger-node-client/api_jenkinsInfo.js.html (hunk 1 above)
Comment (laurafitzgerald) [reply docs/specs/aerogear-digger-node-client/api_jenkinsInfo.js.html]: @aliok thanks. I carried out those steps. Just wondering should these files be linked from somewhere. If they are generated I guess JSDocs knows what it's doing. There is however no way for the user to get to this file without manually adding it in the URL. But if we are happy with this then there is no change required.

File: docs/specs/aerogear-digger-node-client/index.html (hunk 2 above)
Comment (laurafitzgerald) [reply docs/specs/aerogear-digger-node-client/index.html]: :+1:

File: docs/specs/aerogear-digger-node-client/scripts/prettify/Apache-License-2.0.txt (hunk 3 above)
Comment (laurafitzgerald) [reply docs/specs/aerogear-digger-node-client/scripts/prettify/Apache-License-2.0.txt]: :+1:

File: docs/specs/aerogear-digger-node-client/global.html (hunk 4 above)
Comment (laurafitzgerald) [reply docs/specs/aerogear-digger-node-client/global.html]: :+1:

File: docs/specs/aerogear-digger-node-client/util_sslSetup.js.html (hunk 5 above)
Comment (laurafitzgerald) [reply docs/specs/aerogear-digger-node-client/util_sslSetup.js.html]: :+1:

**Comment by laurafitzgerald** [comment]
@aliok Thanks for the response. I didn't go through all the content as (as per the JIRA) this is generated content. The changes relevant to the aerogear.org stuff looks fine to me.

File: docs/specs/aerogear-digger-node-client/api_jenkinsInfo.js.html (hunk 1 above)
Comment (aliok) [reply docs/specs/aerogear-digger-node-client/api_jenkinsInfo.js.html]: we just create a link to JSDoc's index.html. Other pages for the modules are linked from there.

**Pull request merged**